src/js/shield_defs.js

```diff
@@ -4208,6 +4211,45 @@ export function loadShields() {
 
   // Ref-specific cases. Each entry should be documented in CONTRIBUTE.md
 
+  shields["CA:NS:S"].overrideByRef = {
+    BdOLSD: {
+      spriteBlank: "shield_ca_ns_s_bdolsd",
+    },
+    CeT: {
+      spriteBlank: "shield_ca_ns_s_cet",
+    },
+    CT: {
+      spriteBlank: "shield_ca_ns_s_ct",
+    },
+    DNISD: {
+      spriteBlank: "shield_ca_ns_s_dnisd",
+    },
+    ET: {
+      spriteBlank: "shield_ca_ns_s_et",
+    },
+    FdLt: {
+      spriteBlank: "shield_ca_ns_s_fdlt",
+    },
+    GT: {
+      spriteBlank: "shield_ca_ns_s_gt",
+    },
+    KSD: {
+      spriteBlank: "shield_ca_ns_s_ksd",
+    },
+    LR: {
+      spriteBlank: "shield_ca_ns_s_lr",
+    },
+    MD: {
+      spriteBlank: "shield_ca_ns_s_md",
+    },
+    MT: {
+      spriteBlank: "shield_ca_ns_s_mt",
+    },
+    ST: {
+      spriteBlank: "shield_ca_ns_s_st",
+    },
+  };
+
```

Comment:
FSSD appears to be missing:
https://www.openstreetmap.org/relation/11147926

Reply:
The FSSD is documented on Wikipedia but I couldn't find any signage for it.